DirectXTex/DirectXTexImage.cpp

```diff
@@ -20,6 +20,19 @@ namespace DirectX
 
 using namespace DirectX;
 
+#ifndef WIN32
+namespace
+{
+    inline void * _aligned_malloc(size_t size, size_t alignment)
```

Comment:
The Microsoft Standard C++ Library doesn't currently implement this C+17 ``std::aligned_alloc``. That because the C++17 standard version is freed with ``free`` instead of a specific function, and on Windows these are all thin wrappers over the Win32 Heap API.